### ISSUE TITLE
1.12.1 dim 11 just gives up scanning further [java.lang.ThreadGroup:uncaughtException]

### ISSUE THREAD

**Issue by SDUBZ** [way2muchnoise/JustEnoughResources#123]
Forge: 1.12.1-2480
jer: 1.12.1-0.8.4.25

[java.lang.ThreadGroup:uncaughtException:-1]: java.lang.IllegalArgumentException: Could not get provider type for dimension -11, does not exist
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraftforge.common.DimensionManager.getProviderType(DimensionManager.java:146)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraftforge.common.DimensionManager.createProviderFor(DimensionManager.java:296)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at net.minecraft.world.WorldServer.<init>(WorldServer.java:109)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at jeresources.profiling.DummyWorld.<init>(DummyWorld.java:36)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at jeresources.profiling.Profiler.profileWorld(Profiler.java:54)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at jeresources.profiling.Profiler.run(Profiler.java:39)
[12:07:53] [Thread-37/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: 	at java.lang.Thread.run(Unknown Source)

**Comment by way2muchnoise** [comment]
What other mods are installed?

**Comment by SDUBZ** [comment]
let me get you a modlist

**Comment by SDUBZ** [comment]
here ya go http://puu.sh/xAZ6v/5d1f50cf7d.txt

**Comment by way2muchnoise** [comment]
Could you track down what mod adds the bad dimension?

**Comment by SDUBZ** [comment]
i will look further later on but as for now trying to trackdown a mod that crashing pack server side and crash report gives unknown source

**Comment by way2muchnoise** [comment]
All you need to do is find out which mod adds the dimension with id -11

**Comment by SDUBZ** [comment]
the only dim that is -11 in it is the deep dark in exu2 see  I:"Deep Dark"=-11325

**Comment by lurr** [comment]
@way2muchnoise This is Applied Energistics 2's Spatial IO Dimension, as seen in the AE2 config.
```spatialio {
    D:spatialPowerExponent=1.35
    D:spatialPowerMultiplier=1250.0
    I:storageDimensionID=2
    I:storageProviderID=-11
}```

**Comment by StarkRG** [comment]
I am also having this problem. It'd be nice if there was a dimension blacklist.

**Comment by PiggyWhiskey** [comment]
Issue still exists in JER 1.12.2-0.8.5.27

FTB Revelations 1.2.0

Exact same error preventing Profiling unless manually visiting every dimension and messing around with JSON to join the files.

**Comment by way2muchnoise** [comment]
@PiggyWhiskey the issue was fixes in JER 1.12.2-0.8.6+, but #138 is working on a better solutions.
